HEAL-notebooks/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jupyter/minimal-notebook
+
+ARG NOTEBOOK_DIR
+
+COPY $NOTEBOOK_DIR/ $HOME/
+
+RUN printf '\n\
```

Comment:
I totally get why you'd want to do this, but I think it makes code updates tricky in the future. It's a little more harrowing to edit this if needed. I think having a simple bash script and doing a `COPY` and `RUN` might be easier for future devs (at least until Dockerfiles get native multi-line `RUN` command support)